lambdas/handlers/login_redirect_handler.py

```diff
@@ -14,18 +14,11 @@
 @set_request_context_for_logging
 @override_error_check
 @ensure_environment_variables(names=["AUTH_DYNAMODB_NAME"])
+@handle_lambda_exceptions
 def lambda_handler(event, context):
     request_context.app_interaction = LoggingAppInteraction.LOGIN.value
     login_redirect_service = LoginRedirectService()
```

Comment:
Suggest to add an entry point log just for consistency

```suggestion
    login_redirect_service = LoginRedirectService()
    logger.info("Login Redirect handler triggered")
```